preparata.yml

```diff
@@ -0,0 +1,48 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# code id, physical, logical are all lower case
+code_id: preparata
+physical: bits
+logical: bits
+
+name: 'preparata'
+introduced: '\url{https://doi.org/10.1007/0-387-34799-2_39}'
+
+description: 'Preparata code is a \([[2^{m+1}-1, 2^{m+1}-2m-2, 5]]\) binary code where \(m\) is odd. The code is non linear and can exist for perimeters 
+for which there is no linear code. Extended preparata codes can be thought of as characteristic function of the tuple (X,Y) where X, Y belongs to \(\mathbb{F}_{2^m}\) along with the
```

Comment:
"Extended preparata codes can be thought of as characteristic function of the tuple (X,Y) where X, Y belongs to \(\mathbb{F}_{2^m}\) along with the following properties"

This sentence is mathematically incorrect, and grammar is flawed.
with

"Codewords of the extended Preparata code can be